commit 08ae4c6418df2e3ab507d60f81228d350ba26618
Author: Dan Williams <dcbw@redhat.com>
Date:   Thu Mar 9 11:32:28 2017 -0600

    USB: serial: option: add Quectel UC15, UC20, EC21, and EC25 modems
    
    commit 6e9f44eaaef0df7b846e9316fa9ca72a02025d4 upstream.
    
    Add Quectel UC15, UC20, EC21, and EC25.  The EC20 is handled by
    qcserial due to a USB VID/PID conflict with an existing Acer
    device.
    
    Signed-off-by: Dan Williams <dcbw@redhat.com>
    Cc: stable <stable@vger.kernel.org>
    Signed-off-by: Johan Hovold <johan@kernel.org>

Index: linuxtest/dummy/rpi_1470_08ae4c6418df2e3ab507d60f81228d350ba26618.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1470_08ae4c6418df2e3ab507d60f81228d350ba26618.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
